src/core/Expander/Expander/Expander.test.tsx

```diff
@@ -0,0 +1,171 @@
+    </Expander>
+  );
+
+  it('gives the classname to expander title and icon', () => {
+    const { getByTestId } = render(DefaultOpenExpander());
+    const div = getByTestId('expander-open-by-default-title');
+    expect(div).toHaveClass('fi-expander_title--open');
+    expect(div.querySelector('svg')).toHaveClass(
+      'fi-expander_title-icon--open',
+    );
+  });
+
+  it('classnames will be removed when clicked', () => {
+    const mockClickHandler = jest.fn();
+    const { getByTestId, getByRole } = render(
+      DefaultOpenExpander({ onOpenChange: mockClickHandler }),
+    );
+    const buttonToClick = getByRole('button');
+    const titleDiv = getByTestId('expander-open-by-default-title');
+    fireEvent.click(buttonToClick);
+    expect(titleDiv).toHaveClass('fi-expander_title');
+    expect(titleDiv).not.toHaveClass('fi-expander_title--open');
+    expect(buttonToClick.querySelector('svg')).not.toHaveClass(
+      'fi-expander_title-icon--open',
+    );
+  });
+});
+
+describe('onOpenChange', () => {
+  it('is called', async () => {
+    const mockClickHandler = jest.fn();
+    const { getByRole } = render(
+      <Expander onOpenChange={mockClickHandler}>
+        <ExpanderTitle>Test expander open by default</ExpanderTitle>
+        <ExpanderContent>Test expander open by default content</ExpanderContent>
+      </Expander>,
+    );
+    const button = getByRole('button');
+    fireEvent.click(button);
+    expect(mockClickHandler).toHaveBeenCalledTimes(1);
+  });
+});
+
+describe('open', () => {
+  const ControlledExpander = (props?: Partial<ExpanderProps>) => (
+    <Expander className="expander-onclick-test" open={true} {...props}>
+      <ExpanderTitle {...{ 'data-testid': 'expander-title-id' }}>
+        Test expander onClick testing
+      </ExpanderTitle>
+      <ExpanderContent>Test expander click testing content</ExpanderContent>
+    </Expander>
+  );
+
+  it('open-classnames should be found ', async () => {
+    const { getByTestId } = render(ControlledExpander());
+    const div = getByTestId('expander-title-id');
+    expect(div).toHaveClass('fi-expander_title--open');
+
+    expect(div.querySelector('svg')).toHaveClass(
+      'fi-expander_title-icon--open',
+    );
+  });
+
+  it('is clicked. Should not change as it is controlled outside', async () => {
+    const mockClickHandler = jest.fn();
+    const { getByRole, getByTestId } = render(
+      ControlledExpander({ onOpenChange: mockClickHandler }),
+    );
+    const button = getByRole('button');
+    fireEvent.click(button);
+    expect(mockClickHandler).toHaveBeenCalledTimes(1);
+    const div = getByTestId('expander-title-id');
+    expect(div).toHaveClass('fi-expander_title--open');
+
+    expect(button.querySelector('svg')).toHaveClass(
+      'fi-expander_title-icon--open',
+    );
+  });
+});
+
+test('CSS export', () => {
+  const css = cssFromBaseStyles(baseStyles);
+  expect(css).toEqual(expect.stringContaining('background-color'));
+});
```

Comment:
This have been here before this PR, but should we remove it as it seems quite useless.

Reply:
Will remove.

Reply:
Removed